website/docs/docs/dbt-versions/core-upgrade/06-upgrading-to-v1.9.md

```diff
@@ -49,6 +49,9 @@ Starting in Core 1.9, you can use the new [microbatch strategy](/docs/build/incr
 - Simplified query design: Write your model query for a single batch of data. dbt will use your `event_time`, `lookback`, and `batch_size` configurations to automatically generate the necessary filters for you, making the process more streamlined and reducing the need for you to manage these details.
 - Independent batch processing: dbt automatically breaks down the data to load into smaller batches based on the specified `batch_size` and processes each batch independently, improving efficiency and reducing the risk of query timeouts. If some of your batches fail, you can use `dbt retry` to load only the failed batches.
 - Targeted reprocessing: To load a *specific* batch or batches, you can use the CLI arguments `--event-time-start` and `--event-time-end`.
+- [Parallel batch execution](docs/build/incremental-microbatch#parallel-batch-execution): Multiple batches are processed at the same time, instead of one after the other (sequentially) for faster processing of your microbatch models. dbt determines if a batch can run in parallel, so manual configuration is usually unnecessary. However, the `concurrent_batches` config is available as an override (not a gate), allowing you to specify whether batches should or shouldn’t be run in parallel in specific cases.
```

Comment:
changing it so it's clear that this is done by dbt automatically would love @QMalcolm to triple check my interpretation though!
```suggestion
- [Automatic parallel batch execution](docs/build/incremental-microbatch#parallel-batch-execution): Process multiple batches at the same time, instead of one after the other (sequentially) for faster processing of your microbatch models. dbt intelligently auto-runs batches in parallel, while also allowing you to manually override parallel execution with the `concurrent_batches` config.
```